src/plugins/link/test/ts/browser/ForceExternalTargetsTest.ts

```diff
@@ -0,0 +1,47 @@
+import '../../../../../themes/silver/main/ts/Theme';
```

Comment:
This can just be `import 'tinymce/themes/silver/Theme';`

Reply:
Updated in https://github.com/tinymce/tinymce/pull/5021/.